gcs/tests/gcs-sample.yaml

```diff
@@ -0,0 +1,152 @@
+apiVersion: triggers.tekton.dev/v1alpha1
+kind: TriggerTemplate
+metadata:
+  name: gcs-triggertemplate
+spec:
+  params:
+    - name: pvc-name
+    - name: project-name
+    - name: secret-name
+  resourceTemplates:
+    - apiVersion: tekton.dev/v1beta1
+      kind: PipelineRun
+      metadata:
+        name: gcs-pr-$(uid)
+      spec:
+        workspaces:
+        - name: data
+          persistentVolumeClaim:
+            claimName: $(params.pvc-name)
+        - name: credentials
+          secret:
+            secretName: $(params.secret-name)
+        pipelineSpec:
+          workspaces:
+          - name: data
+          - name: credentials
+          tasks:
+          - name: gcs-create-bucket
+            taskRef:
+              name: gcs-create-bucket
+            workspaces:
+            - name: credentials
+              workspace: credentials
+            params:
+            - name: bucketName
+              value: gs://tekton-test-bucket-$(uid)
+            - name: project
+              value: $(params.project-name)
+          - name: create-data
+            runAfter: [gcs-create-bucket]
+            taskSpec:
+              workspaces:
+              - name: data
+              steps:
+              - name: write-data
+                image: ubuntu
+                script: |
+                  #!/usr/bin/env bash
+                  set -xe
+                  mkdir -p $(workspaces.data.path)/$(uid)/cookies/
+                  echo "Test data $(uid)" > $(workspaces.data.path)/$(uid)/cookies/recipe
```

Comment:
put a recipe in the recipe file or call the file something different.

Reply:
now i suddenly want to create something called "tekton recipes"